integrated_channels/api/v1/logs/views.py

```diff
@@ -24,32 +26,65 @@
     LearnerDataTransmissionAudit,
 )
 
-from .serializers import ContentSyncStatusSerializer, GenericLearnerSyncStatusSerializer, LearnerSyncStatusSerializer
+from .serializers import ContentSyncStatusSerializer, LearnerSyncStatusSerializer
 
 LOGGER = logging.getLogger(__name__)
 
 
+class ReportingSyncStatusPagination(PageNumberPagination):
```

Comment:
total number of pages is required for manual pagination so I added it to my own custom pagination